pokemongo_bot/cell_workers/seen_fort_worker.py

```diff
@@ -119,6 +144,16 @@ def work(self):
         sleep(2)
         return 0
 
+    def catch_pokemon(self, pokemon):
+        worker = PokemonCatchWorker(pokemon, self)
```

Comment:
That would break PokemonCatchWorker, see __init__ : 
` self.pokemon_list = bot.pokemon_list`

`self.item_list = bot.item_list`